src/actions/filter.jsx

```diff
@@ -29,6 +30,20 @@ export function changeDateRange( filterName, minDate, maxDate ) {
   }
 }
 
+/**
+ * Notifies the application that date interval (m, w, yr) was toggled
+ *
+ * @param {string} dateInterval which filter was clicked
+ * @returns {string} a packaged payload to be used by Redux reducers
+ */
+export function dateIntervalToggled( dateInterval ) {
+  return {
+    type: DATE_INTERVAL_CHANGED,
+    dateInterval,
+    requery: REQUERY_HITS_ONLY
```

Comment:
For a date __filter__ on the left-side this should be `REQUERY_BOTH` if we are using two queries.